### ISSUE TITLE
Can not import optax

### ISSUE THREAD

**Issue by ankit27kh** [deepmind/optax#456]
Import fails with 
```
    ArrayDType = jax._src.numpy.lax_numpy._ScalarMeta  # pylint: disable=protected-access
AttributeError: module 'jax' has no attribute '_src'
```
Optax version 0.1.3
Jax version 0.3.25
Jaxlib version 0.3.25+cuda11.cudnn82

**Comment by mkunesch** [comment]
Thanks, judging by https://github.com/deepmind/chex/issues/197 all that's required is to increase the chex version requirement in optax. I'll bump it to `0.1.5`.

**Comment by mkunesch** [comment]
This has been merged in #456, commit 543e18. We will release a new version in a moment so that you can specify that in the requirements.

**Comment by mkunesch** [comment]
Version `0.1.4` is now released so starting with this version the error mentioned above should no longer happen. Thanks for raising this!
